api/docs/building.dox

```diff
@@ -45,7 +45,7 @@ To build DynamoRIO on Linux, use the following commands as a guide.  This builds
   ```
   # Install dependencies for Ubuntu 15+.  Adjust this command as appropriate for
   # other distributions (in particular, use "cmake3" for Ubuntu Trusty).
-  $ sudo apt-get install cmake g++ g++-multilib doxygen git zlib1g-dev libunwind-dev libsnappy-dev liblz4-dev
+  $ sudo apt-get install cmake g++ g++-multilib doxygen git zlib1g-dev libunwind-dev libsnappy-dev liblz4-dev libxxhash-dev
```

Comment:
We don't have a section for required packages that are necessary only for tests.
Should I make one or just adding it here is ok?

Reply:
Hmm, mixed feelings: separate section adds complexity; seems ok to just put it here.

Reply:
SGTM.